modules/end-user-guide/pages/devfile-introduction.adoc

```diff
@@ -0,0 +1,25 @@
+[id="introduction-to-devfile_{context}"]
+= Introduction to devfile in {prod}
+
+link:https://devfile.io/[Devfiles] are `yaml` text files used for development environment customization. Use them to configure a devfile to suit your specific needs and share the customized devfile across multiple workspaces to ensure identical user experience and build, run, and deploy behaviours across your team.
```

Comment:
Be careful with links because they take users away from your documentation and you're not sure if they'll come back to your page. If you think linking to devfiles help the user, then keep it :) 

Personal preference, I'm not the biggest fan of the very first word, Devfiles, being a hyperlink. It's almost inviting users to not finish the sentence. 

Consider putting the link at the end of the introduction. Something like, `To learn more about how devfiles, go to (link)<Devfile.io`

Or put this sentence I wrote above under an `Additional Resources` section.

Reply:
I understand where you're coming from but in my opinion, the hyperlink placement is ok. If the users click the link, the page has still done its job by providing useful information. Additionally, there are no marketing goals here so we don't have to worry about keeping the users on the page by all means possible.